fsm/src/main/java/org/egov/fsm/util/FSMErrorConstants.java

```diff
@@ -0,0 +1,76 @@
+package org.egov.fsm.util;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class FSMErrorConstants {
+
+
+	// MDMS
+
+	public static final String INVALID_SEARCH = "INVALID SEARCH";
+	public static final String INVALID_SEARCH_1 = "INVALID SEARCH";
+	public static final String INVALID_ADDRES = "INVALID ADDRESS";
+	public static final String BOUNDARY_ERROR ="BOUNDARY ERROR";
+	public static final String BOUNDARY_MDMS_DATA_ERROR ="BOUNDARY MDMS DATA ERROR";
+	public static final String INVALID_BOUNDARY_DATA ="INVALID BOUNDARY DATA";
+
+	public static final String INVALID_PROPERTY_TYPE ="INVALID_PROPERTY_TYPE";
+
+	public static final String INVALID_APPLICATION_CHANNEL ="INVALID_APPLICATION_CHANNEL";
+	public static final String INVALID_PIT_TYPE ="INVALID_PIT_TYPE";
+	public static final String INVALID_UPDATE ="Invlaid Update";
+	public static final String INVALID_COMMENT_CANCEL_REJECT ="INVALID_COMMENT_CANCEL_REJECT";
+	public static final String INVALID_VEHICLE_TYPE = "INVALID_VEHICLE_TYPE";
+	
+	
+	public static final String INVALID_APPLICANT_ERROR ="INVALID_APPLICANT_ERROR";
+	public static final String INVALID_TRIP_AMOUNT ="INVALID_TRIP_AMOUNT";
+	
+	public static final String RECEIPT_ERROR = "RECEIPT ERROR";

```

Comment:
Use _ seperated error codes instead of space

Reply:
Done